markup/pandoc/convert.go

```diff
@@ -77,6 +85,51 @@ func getPandocBinaryName() string {
 	return ""
 }
 
+var versionOnce sync.Once
+
+// getPandocVersion parses the pandoc version output
+func getPandocVersion(cfg converter.ProviderConfig) (string, error) {
+	var version string
+	var err error
+
+	versionOnce.Do(func() {
+		argsv := collections.StringSliceToInterfaceSlice([]string{"--version"})
```

Comment:
The above works, but it doesn't make any sense when you can just do `argsv := []string{"--version"}`

Reply:
I tried, but it does not work that way:

```go
argsv := []string{"--version"}

var out bytes.Buffer
argsv = append(argsv, hexec.WithStdout(&out))
```
does not compile:
```
markup/pandoc/convert.go:98:25: cannot use hexec.WithStdout(&out) (value of type func(c *hexec.commandeer)) as type string in argument to append
```
It works with StringSliceToInterfaceSlice, but don't ask me why – I adapted the whole function from markup/internal/external.go:
https://github.com/gohugoio/hugo/blob/8e2fd55923a4da38eb2ddda51dc1b96943be0c56/markup/internal/external.go#L23

Reply:
I assume that the `InterfaceSlice` is compatible to contain not only strings but also `func(*hexec.comandeer)`, so a conversion probably does make sense here. But again, I am new to Go, so I might be wrong.

Reply:
Yea, right, it's a slice of `any`, you need to do

```go
argsv := []any{"--version"}
```